### ISSUE TITLE
Facebook Messenger Bot

### ISSUE THREAD

**Issue by joshmedeski** [ezekg/swa-dashboard#20]
Hey Zeke, I just read your article and I love this little project!

Just an idea for you (and anyone else with free time), you could use a Facebook Messenger bot instead of Twilio, you could even teach the bot to place the order (if you decided to get even more sophisticated).

I look forward to playing around with this project for my next flight.

Josh

**Comment by ezekg** [comment]
This is a cool idea. Also Slack integration may be cool.

**Comment by graywolf336** [comment]
Yeah, an integration that can be hooked into Slack, Rocket.Chat, etc would be awesome. I might submit a pull request to add this :thinking:

**Comment by aranaur** [comment]
I'd love to see Telegram support.
